app/controllers/application_controller.rb

```diff
@@ -15,4 +17,10 @@ def configure_permitted_parameters
   def set_user
     UserInfo.current_user = authenticate_user!
   end
+
+private
+
+  def set_locale
```

Comment:
we shouldn't need any of this. We're not passing the locale using `params`. We have a single locale, which is set as default